src/pages/Bridge/RedeemForm/SubmittedRedeemRequestModal/index.tsx

```diff
@@ -76,23 +77,17 @@ const SubmittedRedeemRequestModal = ({
                 )}`}
               </span>
             </div>
-            <div>
+            <div className={clsx('flex', 'items-center', 'justify-center', 'space-x-2')}>
               <label
                 htmlFor={USER_BTC_ADDRESS}
                 className={clsx(
                   { 'text-interlayTextSecondaryInLightMode': process.env.REACT_APP_RELAY_CHAIN_NAME === POLKADOT },
                   { 'dark:text-kintsugiTextSecondaryInDarkMode': process.env.REACT_APP_RELAY_CHAIN_NAME === KUSAMA }
                 )}
               >
-                {t('redeem_page.btc_destination_address')}
+                {t('redeem_page.btc_destination_address')}:
               </label>
-              <span
-                id={USER_BTC_ADDRESS}
-                // TODO: could componentize
-                className={clsx('block', 'p-2.5', 'border-2', 'font-medium', 'rounded-lg', 'text-center')}
-              >
-                {request.userBTCAddress}
-              </span>
+              <AddressWithCopyUI id={USER_BTC_ADDRESS} address={request.userBTCAddress} />
```

Comment:
In the process, I found this case missing from the main "address copy feature" PR so added it although it was out of scope (progressive enhancement).
<img width="1496" alt="Screenshot 2022-12-11 at 4 50 12 AM" src="https://user-images.githubusercontent.com/84005068/206886448-e140f172-f243-469b-8002-2810e34b7989.png">
<img width="1496" alt="Screenshot 2022-12-11 at 4 58 09 AM" src="https://user-images.githubusercontent.com/84005068/206886453-57d84d27-dcb0-44ba-a6c1-568b76b9c7fe.png">

Reply:
Please don't do this — either raise a ticket, or if it's a quick fix include it in a separate PR. We don't want out of scope changes included as it makes it very easy to overlook things during testing. Could you back this out please and handle it as described above?

Reply:
OK, I will create a separate PR.

Reply:
Done: https://github.com/interlay/interbtc-ui/pull/753.